packages/components/tooltip/README.mdx

```diff
@@ -50,3 +50,5 @@ import { Tooltip } from '@contentful/f36-tooltip';
   <TextLink>Hover me</TextLink>
 </Tooltip>
 ```
+
+- When using `ReactElement` as the content, it's recommended to use the `ScreenReaderOnly` component when displaying critical information.
```

Comment:
This paragraph should be part of the "Content Guidlines" area on the element and extended
```suggestion
The Tooltip component allows you to pass React elements as content. However, this should be used with care. React.Element as content is best suited for text formatting purposes. It should not be used for interactive elements like links, buttons, or form elements. Additionally, extensive images should be avoided, except for text-decorator icons that give semantic meaning to the text shown in the tooltip (e.g., warning signs or other relevant icons).

```

Also extend the Accessibility part with something like:

Screen readers cannot access the content of a tooltip, so crucial information should not be placed exclusively within a tooltip. To improve accessibility use the <ScreenreaderOnly> component that contains the same text as the tooltip.